src/screens/account/account.screen.tsx

```diff
@@ -0,0 +1,66 @@
+import { useCallback, useEffect, useState } from 'react';
+import { observer } from 'mobx-react-lite';
+import { Card, Text } from '../../components/primitive';
+import { accountService } from '../../services';
+import { stores } from '../../store';
+import { AccountScreenContainer, Title } from './account.screen.styles';
+
+export const AccountScreen = observer(() => {
+  const login = async () => console.log(await accountService.login());
+
+  return (
+    <AccountScreenContainer>
+      <Title variant='contentHeader' tx='common.account' />
+      <Card>
+            <Text variant='contentHeader' tx='common.profile' />
+            <br />
+            <Text variant='contentHeader' text='Address' />
+            <Text variant='content' text={stores.accountStore.address} />
+            <br />
+            <Text variant='contentHeader' text='Balance' />
+            <Text variant='content' text={stores.accountStore.balance} />
+            <br />
+            <br />
+            <br />
+            <br />
+          
```

Comment:
Using so many break tags is not right. Make use of CSS for spacing.

Reply:
Yeah, sure. This page content was a placeholder. We shall update.